web/lib/components/word/Idioms.jsx

```diff
@@ -1,6 +1,24 @@
+import Link from 'next/link'
 import { Examples } from './Examples'
 import { TextToWords } from './TextToWords'
 
+function IdiomTranslation({ translation }) {
+  if (translation.startsWith('(xem)')) {
+    const [prefix, ...wordToLink] = translation.split(' ')
+
+    return (
+      <>
+        {prefix}&nbsp;
+        <Link href={`/w/${wordToLink.join('-')}`}>
+          <a className="font-bold hover:text-sky-500">{wordToLink.join(' ')}</a>
+        </Link>
+      </>
```

Comment:
Can use this component instead
https://github.com/kcjpop/vd/blob/main/web/lib/components/common/LinkToWord.jsx

Reply:
And use `className="border-b-2 border-sky-600"` so it's similar to here

<img width="387" alt="image" src="https://user-images.githubusercontent.com/172503/162435354-fd2a00b7-8884-45d6-bf03-a112eb2de8bc.png">

Reply:
fixed